SCAutolib/cli_commands.py

```diff
@@ -29,7 +32,8 @@ def cli(ctx, force, verbose, conf):
     logger.setLevel(verbose)
     ctx.ensure_object(dict)  # Create a dict to store the context
     ctx.obj["FORCE"] = force  # Store the force option in the context
-    ctx.obj["CONTROLLER"] = Controller(conf)
+    if ctx.invoked_subcommand and ctx.invoked_subcommand != "gui":
```

Comment:
Hmm interesting, don't you need some kind of conf to prepare the system? Will you do all the preparation manually?

Reply:
Ok, fair enough, but we probably need to run some code from the setup for the GUI to work. I will take a look and identify what commands need to be run.